conda/napari_CellSeg3D_ARM64.yml

```diff
@@ -11,18 +11,14 @@ dependencies:
     - pip:
         - numpy
         - napari>=0.4.14
-        # - opencv-python>=4.5.5
         - scikit-image>=0.19.2
         - matplotlib>=3.4.1
         - tifffile>=2022.2.9
-        # - imageio-ffmpeg>=0.4.5
         - torch>=1.11
         - monai[nibabel, einops]>=0.9.0
         - tqdm
-        # - nibabel
         - scikit-image
-        # - pillow
         - pyclesperanto-prototype
         - tqdm
         - matplotlib
-        # - vispy>=0.9.6
+        - napari_cellseg3d[pyqt5]
```

Comment:
This is not valid, but I can add it

Reply:
We really should let user choose, see https://napari.org/stable/plugins/best_practices.html#don-t-include-pyside2-or-pyqt5-in-your-plugin-s-dependencies

Reply:
As a compromise I added optional reqs for PyQt5/PySide2/PySide6, which you can install as napari-cellseg3d[pyqt5], etc